charts/ocis/templates/authmachine/secret.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if or (not .Values.secretRefs.machineAuthApiKeySecretRef) }}
 {{- $params := (dict)}}
 {{- $_ := set $params "machine-auth-api-key" (randAlphaNum 30) }}
-{{- include "ocis.secret" (dict "scope" . "name" "machine-auth-api-key" "params" $params)}}
+{{- include "ocis.secret" (dict "scope" . "name" (include "secrets.machineAuthAPIKeySecret" .) "params" $params)}}
```

Comment:
Where does this come from. I don't see that a new template for `"secrets.machineAuthAPIKeySecret"` was added here.

Reply:
It's not new, it only was used in a single place before. This change removes duplicated declaration of the constant (the reason why we might have introduced the template in the first place)